lightning/src/ln/channelmanager.rs

```diff
@@ -3559,10 +3581,19 @@ where
 	/// wait until you receive either a [`Event::PaymentFailed`] or [`Event::PaymentSent`] event to
 	/// determine the ultimate status of a payment.
 	///
+	/// # Requested Invoices
+	///
+	/// In the case of paying a [`Bolt12Invoice`], abandoning the payment prior to receiving the
```

Comment:
I think something like this is a bit clearer since users don't pay a B12 invoice directly:
```suggestion
	/// In the case of paying a [`Bolt12Invoice`] via [`Self::pay_offer`], abandoning the payment prior to receiving the
```